check_diff/src/main.rs

```diff
@@ -15,11 +20,36 @@ struct CliInputs {
     rustfmt_config: Option<Vec<String>>,
 }
 
-fn main() {
-    let args = CliInputs::parse();
+/// Clone a git repository and cd into it.
+///
+/// Parameters:
+/// url: git clone url
+/// dest: directory where the repo should be cloned
+fn clone_git_repo(url: &str, dest: &str) {
```

Comment:
Better to make `dest` a `PathBuf`, `&Path`, or `AsRef<Path>` so we can have a stronger typed API instead of relying on strings. You may even consider adding something like the [`url`](https://docs.rs/url/latest/url/index.html) crate to strongly type the repo URL, though maybe that's overkill.

Also, why not have `clone_git_repo` return `Result<()>` to better communicate to callers if the repo was cloned successfully. Additionally, I don't think `clone_git_repo` should be responsible for both cloning the repo and `cd`-ing into the directory.